pkg/api/handlers/files.go

```diff
@@ -282,16 +285,6 @@ func convertFile(file, prefix string) types.File {
 	}
 }
 
-func uploadFile(ctx context.Context, req api.Context, gClient *gptscript.GPTScript, workspaceName string) error {
```

Comment:
This was only used in one spot, so I removed its contents to where it is being used to avoid confusion.